src/lib/components/workflow/workflows-summary-configurable-table/workflows-summary-configurable-table.svelte

```diff
@@ -420,6 +421,10 @@
     @apply z-10 bg-primary w-0 border-r-[3px] border-primary cursor-col-resize;
   }
 
+  .resizer.no-columns-pinned:not(.batch-actions-enabled) {
```

Comment:
I would say just hide the resizer when batch actions are enabled too
```suggestion
  .resizer.no-columns-pinned{
```

but I think the hover state is maybe a bit of a blocker for now. What do you think?

![Screenshot 2023-04-20 at 4 12 01 PM](https://user-images.githubusercontent.com/15069288/233498213-a34591a9-1b43-492b-85ef-8afda90341ef.png)

Reply:
Hm yeah I think just leaving the divider/resizer is OK for now, but I agree that it'd be nicer to hide it an fix the hover state in this case.